src/KubernetesClient/KubernetesClientConfiguration.cs

```diff
@@ -108,5 +108,11 @@ public partial class KubernetesClientConfiguration
         /// Do not use http2 even it is available
         /// </summary>
         public bool DisableHttp2 { get; set; } = false;
+
+        /// <inheritdoc cref="KubernetesJson.AddJsonOptions(Action{JsonSerializerOptions})"/>
+        public void AddJsonOptions(Action<JsonSerializerOptions> configure)
+        {
+            KubernetesJson.AddJsonOptions(configure);
```

Comment:
this is a global var
what about introducing a per kuberclient level json inst? default to KubernetesJson
will use per inst json serializer if `addjsonopt` called

here is some the place where kuberetesjson is being called
https://github.com/kubernetes-client/csharp/blob/aa84b453dca3e4af96137ec60ada5f6fcd6ed694/src/KubernetesClient/Kubernetes.cs#L129

Reply:
Done